github/github-accessors.go

```diff
@@ -10708,6 +10708,70 @@ func (u *UserLDAPMapping) GetURL() string {
 	return *u.URL
 }
 
+// GetCreatedAt returns the CreatedAt field if it's non-nil, zero value otherwise.
+func (u *UserMigration) GetCreatedAt() string {
+	if u == nil || u.CreatedAt == nil {
```

Comment:
is it possible for "u" to be nil?
i don't think you can invoke a function on a nil.

Reply:
Yes indeed it is possible.

Check this out: https://play.golang.org/p/ZgOImESxgz_X

Reply:
Ohh, ok.